### PR TITLE
Fix SelfService Web Dockerfile

### PR DESCRIPTION
**Pull request by woksin**
## Summary

Fix the Dockerfile for SelfService Web

### Fixed

- The Dockerfile for SelfService Web

### CODE REVIEW

**Comment by freshteapot** [comment]
@woksin I added some docs about manually running it via docker (I also wanted to test the dockerfile).

The action is not happy (perhaps you know this)
https://github.com/dolittle/Studio/actions/runs/1166038495

File: Source/SelfService/Web/Dockerfile [hunk 1]
```diff
@@ -1,45 +1,26 @@
-FROM node:14.2.0 AS shared-build
+FROM node:14.17.5 AS shared-build
```
Comment (freshteapot) [inline Source/SelfService/Web/Dockerfile:1]: What if we go crazy and use a more modren node :)

```suggestion
FROM node:16.7.0 AS shared-build
```

> node --version
v16.7.0

**Comment by woksin** [comment]
I know, that's because there are no actions here. The other PR i have will fix this

**Review by freshteapot — APPROVED** [review]
Good stuff, nice work.

**Pull request merged**